.rubocop_todo.yml

```diff
@@ -27,6 +27,9 @@ Metrics/AbcSize:
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
   Max: 130
+  Exclude:
+    - 'app/models/embed/purl.rb'
```

Comment:
Can you move this to an inline exception in the file itself?  Otherwise this will get overwritten if someone regenerates this file.